.github/workflows/sanity.yaml

```diff
@@ -24,4 +24,4 @@ jobs:
       - name: Run linting checks
         uses: "golangci/golangci-lint-action@v2"
         with:
-          version: "v1.43"
```

Comment:
Can we update the version in the makefile as well to match?

Reply:
Actually, I guess its `hack/tools/go.mod`

Reply:
Sure 👍 I updated it and ran `go mod tidy` on the tools go.mod as well